micropython_workshop/project_guide/chapter_1/chapter_1.tex

```diff
@@ -1,2 +1,65 @@
 \chapter{Project 1: Blink}
-% chapter contents
\ No newline at end of file
+
+\section{Overview}
+This project is designed to provide a foundation for subsequent projects in this book (\textbf{and beyond}). Over the course of this project, you will:
+\begin{itemize}
+\item Create a simple circuit using your breadboard
+\item Write a program that runs in a loop
+\item Use MicroPython in your program to interact with your microcontroller's GPIO pins.
+\end{itemize}
+At the end of this project, your microcontroller should run a MicroPython program which alternates a light between its ON and OFF states. Let's get started!
+\begin{figure}[h!]
+\centering
+    \includegraphics[width=.6\linewidth]{chapter_1/success!.jpg}
+    \caption{The end result should look something like this}
+    \label{fig:end_result}
+\end{figure}
+
+\section{Directions}
+
+\subsection{Creating the circuit}
+Using jumper cables, you will be assembling a circuit between your microcontroller, your breadboard, an LED, and a 220\si{\ohm} resistor.
+
+\subsubsection{Attach the microcontroller to the breadboard}
+Carefully insert the pins at the bottom of your microcontroller into the breadboard, making sure that the microcontroller is oriented such that:
+\begin{itemize}
+    \item The pin labeled \textbf{3V3} is inserted in hole at \textbf{Column C, Row 1} of the breadboard (or \textbf{C1}, for short)
+    \item The pin labeled \textbf{Vin} is inserted in hole \textbf{J1} of the breadboard
+    \item The pin labeled \textbf{D0} is inserted in hole \textbf{C15} of the breadboard
+    \item the pin labeled \textbf{A0} is inserted in hole \textbf{J15} of the breadboard
+\end{itemize}
+You may need to apply more pressure than expected to seat the microcontroller properly in the breadboard. When its over, it should look like this:
+\begin{figure}[h!]
+    \centering
+    \includegraphics[width=.6\linewidth]{chapter_1/microcontroller_seated_in_breadboard.jpg}
+    \caption{So far, so good!}
+    \label{fig:microntroller+breadboard}
+\end{figure}
+
+\subsubsection{Connect your microcontroller to the positive and negative rails of your breadboard}
+% Should we have an overview of components before this? Will the girls know what a 'jumper wire' is? Or the layout of the breadboard?
+Using a jumper wire, pin one end of the wire into hole \textbf{A1} of the breadboard and the other end on \textit{any} hole in the column labelled with a \textbf{+} on the side of the breadboard. This column is called the \textbf{positive rail}, and will eventually move power from your microcontroller around the circuit we are building.
```

Comment:
I was following the helpful fritzing diagram as I was writing this. but just realized that connecting the 3v3 pin to the positive rail has nothing to do with the circuit we are building (at least I think its superfluous...I'm a total dummo though). Should we consider leaving this out (and update the diagram) to lessen confusion about the path of the circuit?

Reply:
You're right. It's not needed in this circuit. I suppose it was just my habit. I think we should refine them to be as simple as possible, so we can remove it.